contracts/src/FeeVault.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+interface IHypNativeMinter {
```

Comment:
Just want to point out that this implementation is maintained here rn https://github.com/celestiaorg/hyperlane-ops/tree/main/solidity